thriftpy/parser/parser.py

```diff
@@ -540,7 +540,7 @@ def parse(path, module_name=None, include_dirs=None, include_dir=None,
         with open(urlparse(path).netloc + urlparse(path).path) as fh:
             data = fh.read()
     elif url_scheme == '':
-        with open(path) as fh:
+        with open(path, encoding='utf-8') as fh:
```

Comment:
py2.7 has no keyword argument `encoding`. So maybe it's better to read the file in binary mode and decode it to utf-8.